codechecker_lib/database_handler.py

```diff
@@ -96,6 +97,10 @@ def _create_or_update_schema(self, use_migration=True):
         except sqlalchemy.exc.SQLAlchemyError as alch_err:
             LOG.error(str(alch_err))
             sys.exit(1)
+        except CommandError as cerr:
+            LOG.error("Failed to update or create the database schema.")
```

Comment:
This message can be a bit misleading, because in the issue description the problem is if there is a newer schema in the database and we want to connect to it with an older CodeChecker.
In that case no schema update or creation is done. Might be useful to to print out here that there is a database incompatibility between the used CodeChecker version and the database, and updating your CodeChecker release to a newer should resolve it.